### PR TITLE
Migration balance transfer page

### PR DESCRIPTION
**Pull request by filippoweb3**
Deleted 'how-to' guides already present in support docs, left complementary parts, reorg, minor deletions and additions

### CODE REVIEW

**Comment by filippoweb3** [comment]
Run prettify on the branch, no changes.

**Comment by DrW3RK** [comment]
@filippoweb3 When you run `npx pretty-quick ` it should rearrange the content in the doc. Check my latest commit to this branch

image

File: docs/learn/learn-balance-transfers.md [hunk 1]
```diff
@@ -8,96 +8,78 @@ slug: ../learn-balance-transfers
...
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} there are two main ways to
+transfer funds from one account to another:
 
-By default, Polkadot-JS Apps will use `transfer_keep_alive`, ensuring that the account you send from
-cannot drop below the existential deposit of
-{{ polkadot: 1 DOT :polkadot }}{{ kusama: 0.001666 KSM :kusama }}.
+- `transfer keep-alive` (default option) will not allow you to send an amount that would allow the
+  sending account to be removed due to it going below the
+  [existential deposit](https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-)
+  of {{ polkadot: 1 DOT :polkadot }}{{ kusama: 0.001666 KSM :kusama }}.
```
Comment (profradha) [inline docs/learn/learn-balance-transfers.md]: Please fetch these values from the chain state

**Review by DrW3RK — CHANGES_REQUESTED** [review]
There are a few issues with this document, which is not exactly because of this PR. We need to figure out a better way to present information on this page

- As a new reader to this wiki page, I have to hop on to several links to make a basic transfer
- This page lives on both Polkadot and Kusama guides, but the support articles only cater for Polkadot

I have some ideas on how to improve this page, which I will work on shortly. If there are any ideas from your end, feel free to suggest @filippoweb3

**Comment by filippoweb3** [comment]
@DrW3RK let have a look at your requests. I can open a new issue to address to better reorg this page while still merging this PR. Is that ok for you?

**Comment by DrW3RK** [comment]
@filippoweb3 Yes. You can add the lines of code to fetch on-chain values and then merge this PR

File: docs/learn/learn-balance-transfers.md (hunk 1 above)
Comment (filippoweb3) [reply docs/learn/learn-balance-transfers.md]: Added

**Comment by filippoweb3** [comment]
> @filippoweb3 Yes. You can add the lines of code to fetch on-chain values and then merge this PR
@DrW3RK I did it, can you just check it is right for both DOT and KSM? I checked it but just wanted to be sure I did right.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**